README.md

```diff
@@ -43,9 +43,11 @@ Issues and PRs are welcome in accordance with our [contribution guidelines](http
 
 XMTP provides both `production` and `dev` network environments to support the development phases of your project.
 
-The `production` network is configured to store messages indefinitely. XMTP may occasionally delete messages and keys from the `dev` network, and will provide advance notice in the [XMTP Discord community](https://discord.gg/xmtp).
+The `production` and `dev` networks are not interchangeable. For example, an account (XMTP identity) and messages created on the `dev` network can't be accessed from or moved to the `production` network, and vice versa.
```

Comment:
@mkobetic I added this text above to call out that the networks are not interchangeable per your excellent suggestion! Any thoughts? =) cc @saulmc

Reply:
I think the change capture what I had in mind, but I would also preserve the description of the differences between the two network that was there.

Reply:
Thank you, @mkobetic! Ah yes, I kept that info, just moved it below. =) What do you think of the new sequence of info? I was thinking this sequence expresses the correct priority of info - but perhaps not? =)

Reply:
My bad, I missed that @jhaaaa, it's all there indeed.